docs/self-managed/platform-deployment/kubernetes-helm.md

```diff
@@ -86,6 +86,16 @@ elasticsearch-master-0                          0/1     Pending             0
 elasticsearch-master-1                          0/1     Init:0/1            0          4s
 ```
 
+### Installing the Camunda Helm chart on OpenShift
```

Comment:
Not sure if it's a convention, but this title (like other titles here) is redundant.
We are already on the Helm page under `How to install Camunda 8 using Helm` I believe that there is no need to have the full line again here.

something like this should be fine:
> Install on OpenShift

Reply:
As you pointed out, I was just following what seemed to be the convention :smile: I'm happy to apply your suggestion